barretenberg/cpp/src/barretenberg/stdlib/primitives/plookup/plookup.test.cpp

```diff
@@ -14,15 +14,15 @@ using namespace bb;
 using namespace bb::plookup;
 
 // Defining ultra-specific types for local testing.
-using Builder = UltraCircuitBuilder;
+using Builder = UltraCircuitBuilder; // SIMULATOOOOR
```

Comment:
?

Reply:
was trying to figure out what should be simulated initially while bringing this PR to master